### ISSUE TITLE
Update Towny support.

### ISSUE THREAD

**Issue by LlmDl** [APDevTeam/Movecraft#360]
**Is your feature request related to a problem? Please describe.**
Movecraft uses Towny 0.91.0.0 (or something there-abouts,) as a jar inside of Movecraft to provide Towny support. This old version's API has been long-deprecated and Towny 0.96.7.1 finally dropped support for any deprecated classes/methods in builds older than 0.96.0.0.

**Describe the solution you'd like**
Towny does now have a maven repo:
```
  <repositories>
	<repository>
	    <id>jitpack.io</id>
	    <url>https://jitpack.io</url>
	</repository>
  </repositories>
  <dependencies>
	<dependency>
	    <groupId>com.github.TownyAdvanced</groupId>
	    <artifactId>Towny</artifactId>
	    <version>0.96.7.0</version>
	    <scope>provided</scope>
	</dependency>
  </dependencies>  
```

The deprecated TownyUniverse: `import com.palmergames.bukkit.towny.object.TownyUniverse;` can probably be swapped out with the new `import com.palmergames.bukkit.towny.TownyUniverse;` without requiring very much else (any time you used `TownyUniverse.` just replace it with `TownyUniverse.getInstance().`.

**Comment by TylerS1066** [comment]
Thanks for bringing this to our attention!   When we get around to moving Towny support from the core plugin into it's own addon we will make sure to update this!

**Comment by LlmDl** [comment]
Might want to make the change sooner than later, some of our users are already unable to update their towny version because of this, but thanks.

**Comment by TylerS1066** [comment]
> The deprecated TownyUniverse: `import com.palmergames.bukkit.towny.object.TownyUniverse;` can probably be swapped out with the new `import com.palmergames.bukkit.towny.TownyUniverse;` without requiring very much else (any time you used `TownyUniverse.` just replace it with `TownyUniverse.getInstance().`.

Is there some method for us to support all towny versions between 0.93.1.0 (the newest that supports 1.10.2) and the latest without using multiple modules?

**Comment by LlmDl** [comment]
I'm afraid not. Towny's support on the modern builds goes back as far as 1.12.2, which I am planning to do a back-ported release of 0.97.0.0. Even then 1.12 accounts for less than 4% of Towny installs.

**Comment by TylerS1066** [comment]
Fair enough.

**Comment by TylerS1066** [comment]
With the release of Movecraft-Towny ([GitHub](https://github.com/TylerS1066/Movecraft-Towny), [Spigot](https://www.spigotmc.org/resources/movecraft-towny.90429/)) we've moved the support of the Towny plugin to an external addon.

There are two versions of Movecraft-Towny, a 1.14+ jar which targets Towny v0.96.7.0 and a Legacy jar which targets Towny v0.93.1.0.

As of version 7.0.0, Movecraft supports the addon, however, we do not _yet_ support anything but 1.10.2 to 1.12.2 in this repository.

[Eirik's fork](https://github.com/eirikh1996/Movecraft) which brings 1.13+ support has been updated with the appropriate hooks to support the Movecraft-Towny plugin, but unfortunately we are at his discretion for a release.

I'll be closing this issue seeing as it has been solved as much as we can do so.
